### PR TITLE
Fix View.GitHub command and consolidate Guids/PkgCmdIDList

### PR DESCRIPTION
**Pull request by jcansdale**
This fixes the `View > Other Windows > GitHub` command. The command was simply not being called.

It also removes the reference from `GuidList` to `GlobalCommands` and instead consolidates our solution's constants in `Guids.cs` / `PkgCmdID.cs` (in the `GitHub.Exports` project). This lets us access these constants from anywhere without having to reference `GitHub.VisualStudio`.

### CODE REVIEW

File: src/GitHub.Exports/Settings/PkgCmdID.cs [hunk 1]
```diff
@@ -18,5 +19,10 @@ public static class PkgCmdIDList
         public const int githubCommand = 0x320;
         public const int helpCommand = 0x321;
         public const int blameCommand = 0x500;
```
Comment (jcansdale) [inline src/GitHub.Exports/Settings/PkgCmdID.cs]: This ID is referenced by `OpenPullRequests` but isn't actually defined in the `GitHub.VisualStudio.vsct` file. Probably worth fixing in a different PR.

Comment (jcansdale) [inline src/GitHub.Exports/Settings/PkgCmdID.cs]: Arg this isn't the line I thought I commented on.  I tried to comment on the following line:
```
        public const int openPullRequestsCommand = 0x201;
```

File: src/GitHub.Exports/Settings/PkgCmdID.cs [hunk 2]
```diff
@@ -18,5 +19,10 @@ public static class PkgCmdIDList
```
Comment (jcansdale) [inline src/GitHub.Exports/Settings/PkgCmdID.cs]: comment on addConnectionCommand

File: src/GitHub.Exports/Settings/PkgCmdID.cs [hunk 3]
```diff
@@ -4,6 +4,7 @@ namespace GitHub.VisualStudio
 {
     public static class PkgCmdIDList
     {
+        // IDs defined in GitHub.VisualStudio.vsct
```
Comment (grokys) [inline src/GitHub.Exports/Settings/PkgCmdID.cs]: Adding comment on comment.

File: src/GitHub.Exports/Settings/PkgCmdID.cs [hunk 4]
```diff
@@ -4,6 +4,7 @@ namespace GitHub.VisualStudio
...
         public const int addConnectionCommand = 0x110;
         public const int idGitHubToolbar = 0x1120;
         public const int showGitHubPaneCommand = 0x200;
```
Comment (jcansdale) [inline src/GitHub.Exports/Settings/PkgCmdID.cs]: Commenting on the line with  `showGitHubPaneCommand`.

**Pull request merged**